addons/mail/static/src/components/messaging_menu/messaging_menu.js

```diff
@@ -149,6 +141,10 @@ class MessagingMenu extends Component {
         }
     }
 
+    _onClickDesktopTabButton(ev) {
```

Comment:
don't move this method at the wrong alphabetical place (commit: Move onClickDesktopTabButton)